napalm/ios.py

```diff
@@ -320,59 +320,44 @@ def _check_file_exists(self, cfg_file):
             return True
         return False
 
+    def _expand_interface_name(self, interface_brief):
```

Comment:
Could we cache this? Maybe we could have `self.expand_interface_name_map = {}` initialized when the object is instantiated and this method could first check if the short name is there and if it's not execute the code.

The reason is that I think we could reuse this in some other parts as well. Not sure how much complexity was added in the code when @ande0581 refactored the code to return long interface names instead.